<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpreadViewTableWidget</class>
 <widget class="QWidget" name="SpreadViewTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>689</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">#SpreadViewTableWidget {
background-color: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="GraphTableWidget" name="tableView">
     <property name="styleSheet">
      <string notr="true">QTableView {
background-color: white;
border: 1px solid #C9C9C9;
border-left: 0px;
border-bottom: 1px solid black;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="bottomFrame">
     <property name="styleSheet">
      <string notr="true">* {
color: white;
font: bold 11px;
}

#bottomFrame {
border-top: 1px solid rgba(117,117,117,255);
border-bottom: 1px solid rgba(117,117,117,255);
border-right: 1px solid rgba(117,117,117,255);
border-left: 0px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:, y2:1, 
stop:0 rgb(75,75,75),
stop:1 rgb(60, 60, 60));
}

QPushButton {
border: 0px;
border-image: none;
}

QPushButton:hover {
border: 0px;
border-image: none;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:, y2:1, 
stop:0 rgb(85,85,85),
stop:1 rgb(70, 70, 70));
}

QPushButton:pressed, .QPushButton:checked {
border: 0px;
border-image: none;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:, y2:1, 
stop:0 rgb(65,65,65),
stop:1 rgb(50, 50, 50));
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="showOnlySelectedElementsCheckBox">
        <property name="text">
         <string>Selected only</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filterPatternLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="placeholderText">
         <string>Filter values...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>in</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="VisibleSectionsModel" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="filterPushButton">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Apply filter</string>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/GraphPerspective.qrc">
          <normaloff>:/tulip/graphperspective/icons/16/media-playback-start.png</normaloff>:/tulip/graphperspective/icons/16/media-playback-start.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphTableWidget</class>
   <extends>QTableView</extends>
   <header>GraphTableWidget.h</header>
  </customwidget>
  <customwidget>
   <class>VisibleSectionsModel</class>
   <extends>QComboBox</extends>
   <header>VisibleSectionsModel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/GraphPerspective.qrc"/>
  <include location="../../../../library/tulip-gui/resources/TulipGUIResource.qrc"/>
 </resources>
 <connections/>
</ui>
